### DISCUSSION TITLE
Ubuntu 20.04 successfully installed but unable to run

### DISCUSSION THREAD

**Discussion by SongshGeo** [iiasa/CWatM#30] [Q&A]
I successfully installed CWatM through `pip` following the document. 
However, the command `run_cwatm` is not available and there is a command named `cwatm`, instead. 
When I use `cwatm` command for testing, an error from `cwat_initial.py` is raised: 
```
ModuleNotFoundError: No module named cwatm.hydrological_modules.water_demand'
```

I checked the `cwat_initial.py`, it's strange that I cannot trace either of the modules in IDE. 
On the other hand, everything seems well under the windows OS.

I'm a macOS user and I have to use VMware to use CWatM so I'm testing CWatM under both Windows OS and Linux OS. Is there any plan that the model adapted to Mac?

**Comment by guillaumot** [comment]
Hi,
Thanks for your feedback.
usually, you can run the model from Linux or Windows. Is it working if you open cmd into the directory containing the "cwatm" folder and the file 'cwatm.py', and then type the following command "python cwatm.py" ?

If it works, you should be able then to use the command "python C:/XXX/cwatm3.py C:/YYY/settings.ini -l"
to run the model with the associated settings file.

Let me know.

Luca

**Reply by SongshGeo** [reply]
Hi, 
Thanks for your rep.

There is no file named `cwatm.py` in the folder `cwatm`. Instead, I run `run_cwatm.py` after cmd into the directory, but the same error occurred again. 

Here is my files tree in directory `cwatm`:
```
--hydrological_modules
--management_modules
__init__.py
cwatm_dynamic.py
cwatm_initial.py
cwatm_model.py
metaNetcdf.xml
run_cwatm.py
```

**Comment by guillaumot** [comment]
Hello,
thanks. Yes my version was different but you are right. "python run_cwatm.py" should work if you are in the right directory.

Coming back to your error message: is there a 'water_demand' directory inside hydrological_modules ? And inside 'waterdemand' directory is there a python code called 'water_demand.py' ?

Luca

**Reply by SongshGeo** [reply]
Yep, there is. 
When I open the file `water_demand.py`, my IDE hints me Unresolved reference 'cwatm' in the "import" section. 
How strange.
